### PR TITLE
feat(client): inline docs

### PR DESCRIPTION
**Pull request by millsp**
Originally https://github.com/prisma/prisma/pull/9683

### CODE REVIEW

**Review by Jolg42 — APPROVED** [review]
Nice!

**Comment by Jolg42** [comment]
Merging with my approval only since @aqrln is sick today

File: packages/client/src/runtime/utils/common.ts [hunk 1]
```diff
@@ -65,16 +65,18 @@ export const needNamespace = {
...
       return false
     }
-    if (GraphQLScalarToJSTypeTable[field.type]) {
-      return Boolean(needNamespace[field.type])
+    if (GraphQLScalarToJSTypeTable[fieldType]) {
+      return Boolean(needNamespace[fieldType])
     }
   }
 
+  console.log(fieldType)
```
Comment (aqrln) [inline packages/client/src/runtime/utils/common.ts:78]: Stray `console.log`

Comment (Jolg42) [reply packages/client/src/runtime/utils/common.ts:78]: Indeed, can someone create a PR and remove it?

Comment (Jolg42) [reply packages/client/src/runtime/utils/common.ts:78]: PR to remove at https://github.com/prisma/prisma/pull/10263

**Comment by ImLunaHey** [comment]
Is there a reason this implements comments with three `/` instead of the usual two?
Was really confusing when my comments worked but wouldn't show up in the generated types.

**Pull request merged**